src/validation.cpp

```diff
@@ -2911,11 +2911,10 @@ bool CheckBlockHeader(const CBlockHeader& block, CValidationState& state, bool f
 
 
         // Can't do these since current aux chain id is not known.
```

Comment:
Shouldn't line 2913 be deleted along with the following five comment lines? Line 2913 must, I suppose be referring to the already commented out lines from 2914 to 2918.

Reply:
Yes